atom/common/platform_util_win.cc

```diff
@@ -169,33 +170,298 @@ bool OpenExternal(const GURL& url) {
   return true;
 }
 
+
+// Required COM implementation of IFileOperationProgressSink so we can
+// precheck files before deletion to make sure they can be move to the 
+// Recycle Bin.
+class DelFileProgressSink : public IFileOperationProgressSink {
+  public:
+    DelFileProgressSink();
+    ~DelFileProgressSink(void);
+  private:
+	  ULONG STDMETHODCALLTYPE AddRef(void);
```

Comment:
Unless the style in this code is (void), it hasn't been used in C++ in a LONG time, also dtors by definition have no params, so saying void is redundant.